src/data/WorkshopSchedule.js

```diff
@@ -7,7 +7,7 @@ function getDateObj(day, time) {
 
 const workshopSchedule = [
```

Comment:
For now, let's just remove all of these workshops and leave the array empty. The homepage grabs the upcoming few workshops and displays them, which we don't want since we haven't finalized anything yet.

We'll populate this with an actual schedule later.